src/SprykerEco/Yves/Payone/Theme/default/components/molecules/payone-klarna/payone-klarna.twig

```diff
@@ -0,0 +1,49 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'payone-klarna',
+    tag: 'payone-klarna',
+} %}
+
+{% define data = {
+    form: required,
+    billingAddressData: data.billingAddressData | default({}),
+    customerData: data.customerData | default({}),
+    scriptSrc: 'https://x.klarnacdn.net/kp/lib/v1/api.js',
+} %}
+
+{% set billingAddressData = data.billingAddressData %}
+
+{% define attributes = {
+    'klarna-pay-methods': required,
+    'given-name': billingAddressData['given_name'] | default,
+    'family-name': billingAddressData['family_name'] | default,
+    'email': billingAddressData['email'] | default,
+    'street-address': billingAddressData['street_address'] | default,
+    'postal-code': billingAddressData['postal_code'] | default,
+    'city': billingAddressData['city'] | default,
+    'country': billingAddressData['country'] | default,
+    'phone': billingAddressData['phone'] | default,
+    'date-of-birth': data.customerData['date_of_birth'] | default,
+} %}
+
+{% block body %}
+    <div class="payment-subform payone-form payone-klarna-form payment-form" id="payoneKlarna">
+        <div class="field">
+            {{ form_widget(data.form.payoneKlarna.payMethod, {'attr': {'class': config.jsName ~ '__select-field'}}) }}
+            {{ form_widget(data.form.payoneKlarna.payMethodToken) }}
+        </div>
+    </div>
+
+    {% include molecule('script-loader') with {
+        class: config.jsName ~ '__script-loader',
+        attributes: {
+            src: data.scriptSrc,
+            async: true,
+        },
+    } only %}
+
+    <div id="klarna_container"></div>
```

Comment:
Can we use `config.jsName` instead of `id`?

Reply:
No, it's Klarna API requirement. To load the Klarna widget container id should be passed to the load klarna method.